modules/integration/src/test/scala/scala/cli/integration/NativePackagerTests.scala

```diff
@@ -6,10 +6,10 @@ import scala.util.Properties
 
 class NativePackagerTests extends ScalaCliSuite {
   override def group: ScalaCliSuite.TestGroup = ScalaCliSuite.TestGroup.First
-
-  val helloWorldFileName = "HelloWorldScalaCli.scala"
-  val message            = "Hello, world!"
-  val licencePath        = "DummyLICENSE"
+  override def munitFlakyOK: Boolean = TestUtil.isM1 || TestUtil.isNativeCli || TestUtil.isCI
```

Comment:
it could be, but I don't want to accidentally break the other `.flaky` tests in this file: 
https://github.com/Gedochao/scala-cli/blob/d9055102c6ab55553e784d031969742b3c5419db/modules/integration/src/test/scala/scala/cli/integration/NativePackagerTests.scala#L120-L129

Reply:
ahhh, you're right. I misread it by accident.
Lemme fix it.

Reply:
Done